packages/at_client/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - build: consume changes in at_commons v3.0.35 that enforce lowercase on AtKey
 - build: upgrade dependency at_persistence_secondary_server to v3.0.46
 ## 3.0.54
+- feat: Introduce CommitOp(CommitOperation) to the KeyInfo to describe key update or delete upon sync
```

Comment:
This should be in the 3.0.55 section of the changelog
3.0.54 has already been published https://pub.dev/packages/at_client/changelog

Reply:
This has been resolved in the latest commit